java/test/jmri/jmrit/jython/SampleScriptTest.java

```diff
@@ -119,8 +119,14 @@ public void testTurnoutStatePersistencePy() {
 
         // run shutdown task with blocking
         ((jmri.managers.DefaultShutDownManager)InstanceManager.getDefault(ShutDownManager.class)).setBlockingShutdown(true);
+
+        // Uncomment the line below and you will get the summary
+        // if (1==1) return;
+
         InstanceManager.getDefault(ShutDownManager.class).shutdown();
 
+        if (1==1) return;
```

Comment:
@bobjacobsen 
Note these lines. If you uncomment line 124, you will get the summary. So it seems to me that `ShutDownManager.shutdown()` is the culprit. But I don't know why. Maybe a mockup of the shutdown manager fails?